loops-and-arrays.js

```diff
@@ -5,24 +5,37 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+console.log(names.length);
+for (let i = 0; i < names.length ; i++) {
+    names[i] = names[i] + 'je';
+}
+console.log(names);
 
+//DONE!
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
 
-
-
-
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+for (let i = 0; i <numbers.length ; i++) {
+    if (numbers[i]%2 === 0){
+    numbers[i] = numbers[i] * 2;
+    } else if (numbers[i] % 2 !== 0) {
```

Comment:
In principe staat dit gelijk aan een else zonder if
```suggestion
    } else {
```